c++/nda/CMakeLists.txt

```diff
@@ -60,7 +65,7 @@ list(REMOVE_DUPLICATES LAPACK_LIBRARIES)
 target_link_libraries(blas_lapack INTERFACE ${LAPACK_LIBRARIES})
 target_compile_options(blas_lapack INTERFACE ${LAPACK_LINKER_FLAGS})
 if(LAPACK_LIBRARIES MATCHES "mkl")
-  target_compile_definitions(${PROJECT_NAME}_c PUBLIC NDA_HAVE_GEMM_BATCH)
+  target_compile_definitions(${PROJECT_NAME}_c PUBLIC NDA_USE_MKL)
```

Comment:
This works well, however can we add another line: 
```
target_include_directories(${PROJECT_NAME}_c PUBLIC $ENV{MKLROOT}/include)
```
here within the `if` statement? It can happen (for example on my machine) that it does not find the new dependency header file `mkl.h`, because cmake does not add `MKLROOT/include` automaticall as `C_INCLUDE_PATH`. Does it make sense to add this by default?